### ISSUE TITLE
Anchor tag with href aren't cliccable

### ISSUE THREAD

**Issue by andreafmdev** [glidejs/glide#542]
I'm trying to insert a link inside a card into the "glide__slide" , but are not clickable , it's my config problem ? Can any1 help me ? 

<img width="749" alt="SCREENCODE" src="https://user-images.githubusercontent.com/73602929/105648319-25b57680-5eab-11eb-8171-98e78aeb9c62.PNG">
<img width="503" alt="screencss" src="https://user-images.githubusercontent.com/73602929/105648468-d754a780-5eab-11eb-93d0-6640ddb9c8ec.PNG">
<img width="530" alt="jsscreen" src="https://user-images.githubusercontent.com/73602929/105648470-d9b70180-5eab-11eb-9c21-df7ff8cf3025.PNG">

<img width="602" alt="screen2" src="https://user-images.githubusercontent.com/73602929/105648418-8b096780-5eab-11eb-8ece-291e7f9f4aa5.PNG">

**Comment by cdrn** [comment]
Yeah, I've got the same problem. Guessing this is never going to be resolved?
